client/ionic/src/App.tsx

```diff
@@ -32,6 +33,11 @@ const App: React.FC = () => (
       <IonRouterOutlet>
         <Route path="/about" component={About} exact={true} />
```

Comment:
don't know why gts fix isnt catching/enforcing these differences, perhaps line length issue? @advayDev1

Reply:
which issue specifically?

Reply:
the protect yourself route is being changed by gts fix.

Reply:
I'm not sure what you're talking about.  I see an /about route with no changes?

Reply:
sorry, it starts on line 36

Reply:
We should look at that separately, regarding line length lint config.